modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -8,18 +8,53 @@ This guide contains an overview of the changes in Canvas Kit v9. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+- [Glossary](#glossary)
 - [Codemod](#codemod)
-- [Component Deprecations](#component-deprecations)
+- [Token Updates](#token-updates)
+  - [Depth](#depth)
+- [Component Removals](#component-removals)
   - [Drawer](#drawer)
   - [Layout and Column](#layout-and-column)
-  - [Stack](#stack-hstack-vstack)
+  - [Stack, HStack, and VStack](#stack-hstack-and-vstack)
 - [Component Updates](#component-updates)
-  - [Buttons](#buttons)
-	- [Toast](#toast)
+  - [Button](#button)
+  - [Toast](#toast)
 - [Utility Updates](#utility-updates)
   - [focusRing](#focusring)
-- [Token Updates](#token-updates)
-	- [Depth](#depth)
+
+## Glossary
+
+### Main
+
+Our Main package of Canvas Kit tokens, components, and utilities at `@workday/canvas-kit-react` has
+undergone a full design and a11y review and is approved for use in product.
+
+Breaking changes to code in Main will only occur during major version updates and will always be
+communicated in advance and accompanied by migration strategies.
+
+---
+
+### Preview
+
+Our Preview package of Canvas Kit tokens, components, and utilities at
+`@workday/canvas-kit-preview-react` has undergone a full design and a11y review and is approved for
+use in product, but may not be up to the high code standards upheld in the Main package. Preview is
+analagous to code in beta.
+
+Breaking changes can be deployed to Preview at any time without triggering a major version update,
+though such changes will be communicated in advance and accompanied by migration strategies.
```

Comment:
is it worth noting that breaking changes are unlikely but possible? Also a think about preview components is typically the API *might* change and this is a holding pattern for new components that we eventually want to promote. Might encourage users to use this? Like tables has gone through a review but we have one in main, so the breaking change is that we don't want to deprecate and replace in one fell swoop. So we put things in preview and then wait to promote. Idk if that's worth mentioning

Reply:
All good points, let me see what I can do.